### ISSUE TITLE
This interface to HTML5 document checking is deprecated

### ISSUE THREAD

**Issue by jitendravyas** [callumacrae/gulp-w3cjs#12]
I'm getting this error with every file

`````
content.html Line undefined, Column undefined: This interface to HTML5 document checking is deprecated

``````````

**Comment by thomasdavis** [comment]
Which version of the module are you using?

We have pushed a new version which should fix this problem.

**Comment by jitendravyas** [comment]
I just installed it 1 hour before posting this error. Am I missing any update?

**Comment by callumacrae** [comment]
I haven't updated w3cjs in this module yet; I'll merge the pull request adding it today and do a release. Will post back here when done.

**Comment by callumacrae** [comment]
Fixed in 1.0.0. Thanks to @swillner for doing this :)
